### PR TITLE
test_module: set julia project dir to Oscar of the calling process

### PR DESCRIPTION
**Pull request by aaruni96**
Solves #2414

### CODE REVIEW

File: src/utils/tests.jl [hunk 1]
```diff
@@ -11,13 +11,14 @@ For experimental modules, use [`test_experimental_module`](@ref) instead.
 """
 function test_module(file::AbstractString; new::Bool=true)
   julia_exe = Base.julia_cmd()
+  project_path = split(pathof(Oscar), "/src")[1]
```
Comment (lgoettgens) [inline src/utils/tests.jl]:
```suggestion
  project_path = dirname(dirname(pathof(Oscar)))
```
This should be more robust, e.g. in cases where the path is `/home/myuser/src-code/Oscar.jl/src/Oscar.jl`.

Comment (thofma) [reply src/utils/tests.jl:14]: Should use `joinpath` and not `/`.

Comment (thofma) [reply src/utils/tests.jl]: How about `Oscar.oscardir`?

Comment (benlorenz) [reply src/utils/tests.jl]: Maybe even `dirname(Pkg.project().path)`?
The project may have Oscar developed from a different directory.

Comment (fingolfin) [reply src/utils/tests.jl]: I have used this with good success in various places:
```julia
run(`$(Base.julia_cmd()) --project=$(Base.active_project()) $(ARGS)`)
```

**Review by fingolfin — APPROVED** [review]

**Pull request merged**